meta-dts-distro/recipes-dts/dts/dts/dts

```diff
@@ -17,6 +17,20 @@ BOARD_MODEL="$(dmidecode -s system-product-name)"
 
 SE_credential_file="/etc/cloud-pass"
 
+BIOS_VENDOR="$(dmidecode -s bios-vendor)"
+BIOS_VERSION="$(dmidecode -s bios-version)"
+
+DASHARO_VENDOR="3mdeb"
+DASHARO_NAME="Dasharo"
+
+look_for_dasharo() {
+  if [[ $BIOS_VENDOR == *$DASHARO_VENDOR* && $BIOS_VERSION == *$DASHARO_NAME* ]]; then
+    return 0
+  else
+    return 1
+  fi
+}
+
```

Comment:
About `look_for_dasharo` I left comment in `dasharo-deploy` file. Maybe we should do the same with variables, so add and source `dts-environment.sh` script with common variables?

Reply:
@TomaszAIR Done - https://github.com/Dasharo/meta-dts/pull/25/commits/895a9ce7e8a164f6febc0a4e3958214e5000e7cc and  https://github.com/Dasharo/meta-dts/pull/25/commits/22b1c2c1ec48f722fcfd8901d10439bc1b9ecf4b